src/installer/tests/Assets/TestProjects/ManagedHost/ManagedHost.cs

```diff
@@ -0,0 +1,48 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace ComClient
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            if (args.Length == 0)
+            {
+                throw new Exception("Invalid number of arguments passed");
+            }
+
+            switch (args[0])
+            {
+                case "ijwhost":
```

Comment:
Could you please also add another case where we use `AssemblyLoadContext.Default.LoadFromAssemblyPath` or something similar to load the C++/CLI assembly as a managed assembly?
I know it's not exactly IJWHost test, but we probably don't have coverage for that anywhere else.

Reply:
The tests in runtime do `Assembly.Load` to load the C++/CLI assembly as a managed assembly. I left them there, since I figured they were more runtime than host validation.

https://github.com/dotnet/runtime/blob/fa5dcda23c681031b5d2bd68482702baef80b836/src/tests/Interop/IJW/ManagedCallingNative/ManagedCallingNative.cs#L23

https://github.com/dotnet/runtime/blob/fa5dcda23c681031b5d2bd68482702baef80b836/src/tests/Interop/IJW/NativeCallingManaged/NativeCallingManaged.cs#L23

Reply:
Perfect - ignore my comment then.